tests/src/onhost_tests/pipeline/node/stitching_device_node_test.cpp

```
@@ -0,0 +1,60 @@
+    REQUIRE_FALSE(dai::node::stitching::isDevicePlatformSupported(dai::Platform::RVC2));
+    REQUIRE_FALSE(dai::node::stitching::isDevicePlatformSupported(dai::Platform::RVC3));
+    REQUIRE(dai::node::stitching::isDevicePlatformSupported(dai::Platform::RVC4));
+}
+
+TEST_CASE("Stitching accepts deserialized properties", "[Stitching]") {
+    auto properties = std::make_unique<dai::StitchingProperties>();
+    properties->mode = dai::StitchingProperties::Mode::PLANAR_PROJECTION;
+    properties->maxViewWidth = 1280;
+    properties->numInputs = 2;
+
+    auto stitching = std::make_shared<dai::node::Stitching>(std::move(properties));
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)